.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,46 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 */12 * * *'
```

Comment:
Does it work at `00:00`? I'm not sure, see https://crontab.guru/#0_*/12_*_*_*

Reply:
It worked at midnight:

> ![image](https://user-images.githubusercontent.com/2071331/225900237-9eaeb117-c7b8-4ebb-acc0-af80594af7c2.png)

> Triggered via schedule 12 hours ago

https://github.com/pestphp/pest/actions/runs/4442792042

Reply:
👍